### PR TITLE
Add touchpoint flag

### PR DESCRIPTION
**Pull request by austinscreations**
added a flag so touchpoint data is only printed on serial in teh debug env - just as a potential security thing if your using keypad

cleaned up a spelling mistake for json schema

and cleaned up the serial debug so it can look a little nicer. and will get slightly better still switching to the wt32 library that i've turned the wt32 class into. (but i'll do this in a different PR)
also increased serial begin delay to give the serial monitor a chance to catch up.

edit: also not sure why github says there is a conflict...

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -1618,8 +1620,8 @@ void publishBacklightEvent(int brightness)
   {
     // Start serial and let settle
     Serial.begin(SERIAL_BAUD_RATE);
-    delay(1000);
-    Serial.println(F("[wpan] starting up..."));
+    delay(2000);
+    Serial.println(F("[TP32] starting up..."));
```
Comment (sumnerboy12) [inline src/main.cpp:1624]: the OXRS standard for these log prefixes is all lowercase

File: src/main.cpp [hunk 2]
```diff
@@ -479,10 +479,12 @@ void publishBacklightEvent(int brightness)
     data->point.y = ts.y;
     data->state = LV_INDEV_STATE_PR;
 
-    Serial.print("Data x,y ");
+#if defined(TOUCHDEBUG)
+    Serial.print("[TP32] Data x,y ");
```
Comment (sumnerboy12) [inline src/main.cpp:483]: Wrap in `F()`

File: src/main.cpp [hunk 3]
```diff
@@ -1633,10 +1635,9 @@ void publishBacklightEvent(int brightness)
     // start lvgl
     lv_init();
     lv_img_cache_set_size(10);
-    String LVGL_Arduino = "Hello Arduino! ";
+    String LVGL_Arduino = "[TP32] LVGL starting ";
```
Comment (sumnerboy12) [inline src/main.cpp:1638]: Convert this to 
```
Serial.print(F("[tp32] lvgl starting v"));
Serial.print(lv_version_major());
Serial.print(F("."));
Serial.print(lv_version_minor());
Serial.print(F("."));
Serial.println(lv_version_patch());
```
Try and avoid using `String` wherever possible.

**Review by sumnerboy12 — CHANGES_REQUESTED** [review]

File: src/main.cpp (hunk 3 above)
Comment (sumnerboy12) [reply src/main.cpp]: Yeah I know - but if you are changing it, it is a good chance to fix it up properly.

File: platformio.ini [hunk 4]
```diff
@@ -84,6 +84,7 @@ build_flags =
 build_flags = 
   ${env.build_flags}
   -DFW_VERSION="DEBUG-ETH"
+  -DTOUCHDEBUG
```
Comment (sumnerboy12) [inline platformio.ini:87]: Sorry to be pedantic - might be better to call this `DEBUG_TOUCH` so we can add more `DEBUG_XXX` flags later in a consistent naming scheme?

**Review by sumnerboy12 — APPROVED** [review]
Looks good, aside from the one suggestion

File: platformio.ini (hunk 4 above)
Comment (austinscreations) [reply platformio.ini:87]: sure, couldn't hurt

**Review by sumnerboy12 — APPROVED** [review]

**Comment by austinscreations** [comment]
redone and merged in #7

**Pull request closed without merging**